README.md

```diff
@@ -27,10 +84,10 @@
 
 - [bson](https://github.com/mongodb/bson-rust) - rust bson crate
 
-### Contribution
+## Contribution
```

Comment:
Revert, as this is part of `## other` section

Reply:
My bad, yeah that's a typo on my part

Reply:
https://github.com/denodrivers/mango/pull/15/commits/5e0e3d9d15d235bb4aca644f1f3e65e2bcd200d9